tests/non-pytest/mypy-ignore-tests/transfer_data.py

```diff
@@ -0,0 +1,24 @@
+import uuid
+
+from globus_sdk import TransferClient, TransferData
+
+# simple usage, ok
+tc = TransferClient()
+TransferData(tc, "srcep", "destep")
+
+# can set sync level
+TransferData(tc, "srcep", "destep", sync_level=1)
+TransferData(tc, "srcep", "destep", sync_level="exists")
+# unknown int values are allowed
+TransferData(tc, "srcep", "destep", sync_level=100)
+# unknown str values are rejected (Literal)
+TransferData(tc, "srcep", "destep", sync_level="sizes")  # type: ignore[arg-type]
```

Comment:
shouldn't this line be raising a ValueError?

Reply:
It would, you're correct. See below 👇 . I'm going to mark this resolved, but we can discuss more.